packages/nextra-theme-docs/src/utils/normalize-pages.ts

```diff
@@ -225,11 +222,12 @@ export function normalizePages({
         underCurrentDocsRoot: underCurrentDocsRoot || isCurrentDocsTree,
         pageThemeContext: extendedPageThemeContext
       })
+    const title = extendedMeta.title || (type !== 'separator' && a.name)
 
     const getItem = (): Item => ({
       ...a,
       type,
-      title: title || (type === 'separator' ? undefined : getTitle(a.name)),
+      ...(title && { title }),
```

Comment:
I remove `title` package since we no need it here
Example: `getTitle` will convert page name `getting-started` to `Getting-Started`